tests/System.Text.Primitives.Tests/Encoding/Utf8UtilTests.Validation.cs

```diff
@@ -296,7 +296,7 @@ private static void AssertIsInvalidThreeByteSequence(byte[] invalidSequence)
         {
             Assert.Equal(3, invalidSequence.Length);
 
-            byte[] KNOWN_GOOD_BYTES = DecodeHex(EURO_SYMBOL);
+            byte[] KNOWN_GOOD_BYTES = TestHelper.DecodeHex(EURO_SYMBOL);
```

Comment:
nit for constants: use pascal casing

Reply:
Existing unit tests are out of scope of this review. I'll handle it in a separate PR.